src/lib/t.js

```diff
@@ -0,0 +1,3 @@
+const {hexToRgb} = require('color-name-list/scripts/lib');
+
+console.log(hexToRgb('#5ba8ff'))
```

Comment:
we should remove that

Reply:
you see old commit

Reply:
oh you might squash commits before doing a pull request